packages/terra-table/src/Table.module.scss

```diff
@@ -7,211 +6,13 @@
     border-spacing: 0;
     max-width: 100%;
     width: 100%;
-
-    caption {
-      padding-bottom: 0.5em;
-    }
-
-    tr {
-      display: table-row;
-      margin-bottom: 0;
-    }
-
-    td {
-      border: 0;
-      border-left: var(--terra-table-cell-border-left, 0);
-      display: table-cell;
-      text-align: left;
-      vertical-align: top;
-
-      &:first-of-type {
-        border-left: var(--terra-table-first-cell-border-left, 1px solid #c8cacb);
-      }
-
-      &:last-of-type {
-        border-right: var(--terra-table-last-cell-border-right, 1px solid #c8cacb);
-      }
-    }
-
-    th {
-      border: 0;
-      border-left: var(--terra-table-header-cell-border-left, 0);
-      display: table-cell;
-      text-align: left;
-      vertical-align: top;
-
-      &:first-of-type {
-        border-left: var(--terra-table-first-header-cell-border-left, 1px solid #c8cacb);
-      }
-
-      &:last-of-type {
-        border-right: var(--terra-table-last-header-cell-border-right, 1px solid #c8cacb);
-      }
-    }
-
-    thead {
-      background-color: var(--terra-table-thead-background-color, #dedfe0);
-      display: table-header-group;
-
-      tr {
-        border-bottom: var(--terra-table-thead-tr-border-bottom, 0);
-        border-top: var(--terra-table-thead-tr-border-top, 1px solid #c8cacb);
-      }
-
-      th {
-        color: var(--terra-table-thead-th-color, #1c1f21);
-        font-size: var(--terra-table-thead-th-font-size, 1rem);
-        font-weight: var(--terra-table-thead-th-font-weight, bold);
-        vertical-align: bottom;
-      }
-    }
-
-    tbody {
-      background-color: var(--terra-table-tbody-background-color);
-
-      tr {
-        border-top: var(--terra-table-row-border-top, 1px solid #c8cacb);
-
-        &:last-of-type {
-          border-bottom: var(--terra-table-last-row-border-bottom, 1px solid #c8cacb);
-        }
-      }
-    }
-
-    tfoot {
-      tr {
-        border-bottom: var(--terra-table-tfoot-tr-border-bottom, 1px solid #c8cacb);
-        border-top: var(--terra-table-tfoot-tr-border-top, 0);
-        text-align: left;
-      }
-    }
-  }
-
-  .subheader {
-    background-color: var(--terra-table-subheader-background-color, #c8cacb);
-    color: var(--terra-table-subheader-color, #1c1f21);
-    font-size: var(--terra-table-subheader-font-size, 1rem);
-    font-weight: var(--terra-table-subheader-font-weight, bold);
-    text-align: left;
-  }
-
-  .sort-indicator {
-    color: var(--terra-table-sort-indicator-color, #1c1f21);
-    padding-left: 0.7142857143rem;
-
-    > svg {
-      width: var(--terra-table-sort-indicator-width);
-    }
-  }
-
-  .min-width-tiny {
-    min-width: var(--terra-table-cell-tiny-min-width, 3rem);
   }
 
-  .min-width-small {
-    min-width: var(--terra-table-cell-small-min-width, 4.25rem);
+  caption {
```

Comment:
Where is this style applied?

Reply:
I'm not sure, I didn't want to delete it since it was in the existing table. If anyone knows it'd be helpful.

Reply:
Okay, because if it doesn't have a class associated with it passed by Table or any sub-component that pulls this style in, I dont thin it will ever be used with how the class names are localized during webpack

Reply:
Removing unused caption as per offline discussion.